docs/SQLModelingStandard.md

```diff
@@ -76,9 +76,8 @@ infrastructure will be re-designed to follow this guideline.*
   be named like in `OriginCountryID` and `DestinationCountryID`.*
 - Both **primary key and foreign key** fields should end with **`ID`**
   in upper case.  
-- **Date** fields should not be named “Date”. A
-  **qualifier** like “BoughtDate” or “DateAccepted” should be added as
-  required.  
+- **Date** fields must have **qualifers** (i.e. they should not be named “Date”). A
```

Comment:
I like it, and it is not in the negative tone :+1:

Reply:
typo: `qualifer`